src/list-grid.scss

```diff
@@ -0,0 +1,261 @@
+    'warning': ('backgroundColor': var(--sapWarningBorderColor)),
+    'neutral': ('backgroundColor': var(--sapNeutralBorderColor))
+  );
+
+  @include fd-reset();
+  @include fd-flex(column);
+
+  &__item {
+    @include fd-flex(column);
+    @include fd-fiori-focus();
+
+    background-color: var(--sapList_Background);
+    border-radius: $fd-grid-list-border-radius;
+    box-shadow: $fd-grid-list-item-box-shadow;
+    height: 100%;
+    position: relative;
+
+    &--body {
+      @include fd-flex(column);
+
+      height: 100%;
+      justify-content: space-between;
+
+      &--padding {
+        padding: 1rem;
+      }
+    }
+
+    &--counter {
+      color: var(--sapContent_MarkerTextColor);
+    }
+
+    .#{$block}__radio--input {
+      @include fd-focus() {
+        + .#{$block}__radio--label {
+          outline-offset: 0;
+        }
+      }
+
+      + .#{$block}__radio--label {
+        @include fd-empty() {
+          padding: 0;
+        }
+
+        &::after {
+          top: 0;
+          right: 0;
+          bottom: 0;
+          left: 0;
+          margin: auto !important;
+        }
+      }
+    }
+
+    .#{$block}__checkbox--input {
+      @include fd-focus() {
+        + .#{$block}__checkbox--label {
+          outline-offset: 0;
+        }
+      }
+
+      + .#{$block}__checkbox--label {
+        @include fd-empty() {
+          padding: 0;
+        }
+      }
+    }
+
+    .#{$block}__checkbox--label,
+    .#{$block}__radio--label {
+      display: none;
+    }
+
+    &--state-unread {
+      .#{$item}--body * {
+        font-weight: bold;
+      }
+    }
+
+    &--state-actions {
+      margin-top: 1rem;
+    }
+
+    &--link {
+      cursor: pointer;
+
+      @include fd-hover() {
+        box-shadow: var(--fdGrid_List_Item_Hover_BoxShadow);
+        background-color: $fd-grid-list-item-hover-background-color;
+      }
+
+      @include fd-active() {
+        background-color: $fd-grid-list-item-active-background-color;
+
+        * {
+          color: $fd-grid-list-active-text-color;
+        }
+      }
+
+      @include fd-navigated() {
+        &::after {
+          content: "";
+          position: absolute;
+          top: 0;
+          right: 0;
+          height: 100%;
+          width: 0.1875rem;
+          display: block;
+          border-radius: 0 $fd-grid-list-border-radius $fd-grid-list-border-radius 0;
+          background-color: var(--sapList_SelectionBorderColor);
+        }
+      }
+    }
+  }
+
+  .#{$block}__toolbar {
+    @include fd-flex-vertical-center();
+
+    padding: 0 1rem;
+  }
+
+  .#{$block}__group-header {
+    @include fd-flex-vertical-center();
+    @include fd-ellipsis();
+
+    font-size: var(--sapFontHeader5Size);
+    padding-top: 0.75rem;
+    margin-top: 1rem;
+    height: 2.75rem;
+    color: var(--sapList_GroupHeaderTextColo);
+    font-weight: bold;
+    line-height: 2rem;
+
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+
+  &__mode {
```

Comment:
<img width="474" alt="Screen Shot 2020-11-12 at 12 22 42 PM" src="https://user-images.githubusercontent.com/39598672/98973464-cc5fbd00-24e1-11eb-992e-7ccda8aca42f.png">

You can't have a structure like this. Following BEM:
`fd-grid-list` is the **Block**. You can't have a **Block** AND an **Element** `fd-grid-list__mode` on the same level, meaning you can't have `fd-grid-list fd-grid-list__mode`. The **Element** (the one that you add `__` in the name) is a child of the Block. The modifier (`--` in the name) is the one you can use with Blocks and Elements. So you can have `fd-grid-list fd-grid-list--dark-mode` for example, but you can't have `fd-grid-list fd-grid-list__mode`.

Reply:
And you will need a reset if it's an **Element**

Reply:
fixed

Reply:
thank you for the comment :slightly_smiling_face: 
@InnaAtanasova